crates/holochain/src/core/workflow/incoming_dht_ops_workflow.rs

```diff
@@ -39,13 +46,16 @@ impl Default for InOpBatch {
     }
 }
 
-static IN_OP_BATCH: Lazy<parking_lot::Mutex<HashMap<DbKind, InOpBatch>>> =
-    Lazy::new(|| parking_lot::Mutex::new(HashMap::new()));
```

Comment:
take it it didn't work having this be a static anymore?

Reply:
Yeh it causes problems in our tests because different conductors can have the same dnahash